emails/templates/email-verification-reminder.js

```diff
@@ -0,0 +1,61 @@
+const { action, base, link, vspace } = require('../template-utils')
```

Comment:
Please feel free to adjust some of the text if you have any ideas :).

I made this under the assumption that the reminder would contain a new verification JWT, since we have fairly long-lived ones, but wasn't sure after the discussion today.

Reply:
Added small text changes to be more distinct form verification email, so users don't get confused in their inbox :)